gensrc/script/vectorized/vectorized_functions.py

```diff
@@ -53,6 +53,7 @@
     [10132, "round", "DOUBLE", ["DOUBLE", "INT"], "MathFunctions::round_up_to"],
     [10133, "dround", "DOUBLE", ["DOUBLE", "INT"], "MathFunctions::round_up_to"],
     [10134, "truncate", "DOUBLE", ["DOUBLE", "INT"], "MathFunctions::truncate"],
```

Comment:
How about leave a deprecated comment the `truncate` function?

Reply:
Since we provide a new function named `truncate_decimal`, so the original `truncate` is not deprecated.